pkg/version/version.go

```diff
@@ -1,4 +1,57 @@
 package version
 
+import (
+	"fmt"
+	"regexp"
+
+	version "github.com/hashicorp/go-version"
+	coreops "github.com/portworx/sched-ops/k8s/core"
+)
+
 // Version will be overridden with the current version at build time using the -X linker flag
 var Version string
+
+var (
+	kbVerRegex = regexp.MustCompile(`^(v\d+\.\d+\.\d+)(.*)`)
+)
+
+// RegisterV1CRD returns true of k8s version is greater than 1.16
+func RegisterV1CRD() (bool, error) {
```

Comment:
The function name indicates that it will Register a V1 version of CRD. Can we rename this to `IsV1RegistrationNeeded` or `RequiresV1Registration ?`